staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/marshal.go

```diff
@@ -45,6 +45,7 @@ func (s *JSONSchemaPropsOrBool) UnmarshalJSON(data []byte) error {
 		if err := json.Unmarshal(data, &sch); err != nil {
 			return err
 		}
+		nw.Allows = true
```

Comment:
This mistake saved us 🎉

Because for `additionalProperties: {...}` we forgot to set `allows: true` (and we reject `Allows == false` in general), it was not possible to define a CRD with anything other than `additionalProperties: true`.